src/__main__.py

```diff
@@ -51,53 +58,85 @@ def get_code_duplication_percentage(directory_path):
     ]
 
 
-def get_code_duplication_by_range_date(
-    directory_path,
+def get_report_of_code_lines(exclude_test_files=False):
```

Comment:
No es muy aconsejado de tener una función que cambia su comportamiento basado en un booleano.
Ver estos artículos:
- https://martinfowler.com/bliki/FlagArgument.html
- https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3
- https://understandlegacycode.com/blog/what-is-wrong-with-boolean-parameters/
- etc

Seguramente el 1er artículo puede ser el mejor, siendo de Martin Fowler, el gúru de la refactorización.